src/common/translation/i18n/en.json

```diff
@@ -344,6 +344,12 @@
     },
     "ticketmasterNotice": "Registration for the event takes place in the Ticketmaster ticket service. To reserve tickets you will need a user account which you can create during the reservation process if you do not already have one. The up to date status for placing can be found from Ticketmaster."
   },
+  "ticketmasterEvent": {
+    "description": "",
+    "passwordLabel": "",
+    "continueButton": "",
+    "participantsPerInviteText": ""
+  },
```

Comment:
Yeah of course those are needed before this goes live, but they aren't available yet. Personally I prefer leaving missing translations empty rather than filing them with some half-baked DIY versions I will then forget to update 😄